modules/topicsFpdModule.js

```diff
@@ -0,0 +1,65 @@
+import {logError, mergeDeep} from '../src/utils.js';
+import {getRefererInfo} from '../src/refererDetection.js';
+import {submodule} from '../src/hook.js';
+
+export const TOPICS_TAXONOMY = 600;
```

Comment:
this may change over time, tax 1 is 600, if we see future taxonomy versions beyond one, let's do formula 599 + taxonomy version. I'll reserve 601-609 with the IAB now

Reply:
https://github.com/InteractiveAdvertisingBureau/openrtb/pull/109

Reply:
Is there an agreed upon version scheme from topics? how do you propose this should be implemented? 

`taxonomyVersion` is a string that is apparently "1" atm, at some point it could come back as "1.1" "2" or anything, should the code just throw an error if it's not "1"? assign 601 hoping we can notice before the third one comes? Who and how is going to look for changes?

Reply:
let's just assign 600 if it is "1" and abort otherwise. We can modify later if future returns get assigned future id's